com.abagames.noiz2sa.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.abagames.noiz2sa
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```

Comment:
you might as well update the runtime to `21.08`. 20.08 will be EOL in August.

Reply:
It seems to run with 23.08 https://github.com/flathub/com.abagames.noiz2sa/pull/2, please try 23.08.